src/app/article/article.component.ts

```diff
@@ -1,35 +1,33 @@
-import { Component, OnInit } from '@angular/core';
```

Comment:
We talked about this lint issue.
<img width="650" alt="image" src="https://github.com/user-attachments/assets/41b20a60-3205-433c-9316-cc05b967a1c7">

Reply:
Corrected.